### PR TITLE
WIP - DO NOT MERGE Hotfix/5.3.9

### PR DESCRIPTION
**Pull request by drjsanger**
@keiranmraine @sb43 
Please review.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,22 +1,36 @@
 # CHANGES
 
+## v5.3.9
+
+* Change max reads read when determining read length to 50k
+* Modify _get_bam_header_data to get the max read length of ALL bam files encountered and therefore to produce the correct lib_size
+* Converted internal SAM flag constants to point to `Bio::DB::SAM` constants (in the future  internal constants could be replaced)
```
Comment (keiranmraine) [inline CHANGES.md]: `Bio::DB::SAM`? should be `Bio::DB::HTS`

File: perl/bin/cgpVaf.pl [hunk 2]
```diff
@@ -186,6 +186,8 @@ sub option_builder {
                 'pid|id_int_project=s' => \$options{'pid'},
                 'exp|exonerate_pct=i' => \$options{'exp'},
                 'vcf|vcf_files=s{,}' => \@{$options{'vcf'}},
+				'f|filter_inc=i' => \$options{'finc'},
```
Comment (keiranmraine) [inline perl/bin/cgpVaf.pl]: Hard tabs

File: perl/lib/Sanger/CGP/Vaf/Data/ReadVcf.pm [hunk 3]
```diff
@@ -899,14 +898,17 @@ Inputs
 sub _get_read_length {
 	my ($self,$sam)=@_;
 	my ($mapped_length,$read_counter);
-	my $iterator  = $sam->features(-iterator=>1);
+	my $iterator  = $sam->features(	-iterator=>1, 
+									-type=>"match");
```
Comment (keiranmraine) [inline perl/lib/Sanger/CGP/Vaf/Data/ReadVcf.pm]: hard tabs

File: perl/t/vaf.t [hunk 4]
```diff
@@ -18,9 +18,9 @@ use File::Path;
 use Const::Fast qw(const);
 use File::Path qw(mkpath);
 
-const my $MODULE1 => ' Sanger::CGP::Vaf::Data::ReadVcf';
-const my $MODULE2 => 'Sanger::CGP::Vaf::Process::Variant';
-const my $MODULE3 => 'Sanger::CGP::Vaf::VafConstants';
+const my @MODULES => qw( Sanger::CGP::Vaf::Data::ReadVcf
+						Sanger::CGP::Vaf::Process::Variant
```
Comment (keiranmraine) [inline perl/t/vaf.t]: hard tabs

File: perl/lib/Sanger/CGP/Vaf/Data/ReadVcf.pm (hunk 3 above)
Comment (drjsanger) [reply perl/lib/Sanger/CGP/Vaf/Data/ReadVcf.pm]: Argh... new editor

File: perl/t/vaf.t (hunk 4 above)
Comment (drjsanger) [reply perl/t/vaf.t]: Hard tabs all corrected... @keiranmraine

Comment (drjsanger) [reply perl/t/vaf.t]: Changes.md fixed.

**Comment by sb43** [comment]
All new changes looks good to me

**Comment by drjsanger** [comment]
Closing and will hotfix now.

**Pull request closed without merging**